azure/scope/cluster.go

```diff
@@ -208,10 +210,11 @@ func (s *ClusterScope) LBSpecs() []azure.LBSpec {
 				},
 			},
 		},
-		Type:            infrav1.Public,
-		SKU:             infrav1.SKUStandard,
-		BackendPoolName: s.OutboundPoolName(azure.GenerateControlPlaneOutboundLBName(s.ClusterName())),
-		Role:            infrav1.ControlPlaneOutboundRole,
+		Type:                 infrav1.Public,
+		SKU:                  infrav1.SKUStandard,
+		BackendPoolName:      s.OutboundPoolName(azure.GenerateControlPlaneOutboundLBName(s.ClusterName())),
+		Role:                 infrav1.ControlPlaneOutboundRole,
+		IdleTimeoutInMinutes: to.Int32Ptr(4),
```

Comment:
does the default value not apply here?

Reply:
no, there is no spec for control plane outbound LB currently